tests/units/common/test_enhancer.py

```diff
@@ -60,6 +60,128 @@
     "record_creation_date": "2022-05-20T00:00:00",
     "titles": [{"title": "", "subtitle": "", "source": publisher}],
 }
+input_with_affiliation_of_cooperation_agreement_with_CERN = {
+    "abstract": "this is abstracts",
+    "authors": [
+        {
+            "affiliations": [
+                {
+                    "country": "Switzerland",
+                    "organization": "International laboratory covered by a cooperation agreement with CERN",
+                    "value": "International laboratory covered by a cooperation agreement with CERN, Geneva, Switzerland",
+                }
+            ],
+            "email": "test@email.com",
+            "full_name": "Test Surname, Test names",
+            "given_names": "Test names",
+            "surname": "Test Surname",
+            "orcid": "Test Id",
+        }
+    ],
+    "copyright_holder": "copyright_holder",
+    "copyright_year": "2020",
+    "copyright_statement": "copyright_statement",
+    "copyright_material": "copyright_material",
+    "date_published": "2022-05-20",
+    "title": "title",
+    "subtitle": "subtitle",
+}
+expected_output_with_affiliation_of_cooperation_agreement_with_CERN = {
+    "abstracts": [{"value": "this is abstracts", "source": publisher}],
+    "authors": [
+        {
+            "affiliations": [
+                {
+                    "organization": "International laboratory covered by a cooperation agreement with CERN",
+                    "value": "International laboratory covered by a cooperation agreement with CERN, Geneva, Switzerland",
```

Comment:
we are not testing the case that `country` already populated from the XML files.